### PR TITLE
Link PMULL with CE

### PR DESCRIPTION
**Pull request by jforissier**
Assume we can use 64-bit PMULL when the Crypto Extensions config is enabled. Platforms can override this setting if need be.

### CODE REVIEW

File: core/crypto.mk [hunk 1]
```diff
@@ -52,6 +52,12 @@ ifeq ($(CFG_CRYPTO_WITH_CE),y)
 
 $(call force,CFG_AES_GCM_TABLE_BASED,n,conflicts with CFG_CRYPTO_WITH_CE)
 
+# CFG_HWSUPP_PMULL_64 defines whether AArch64 supports polynomial multiplies
```
Comment (jenswi-linaro) [inline core/crypto.mk]: `pmull_ghash_update_p64()` can be available in AArch32 too

Comment (jforissier) [reply core/crypto.mk]: Right. I'll rename/reword the stuff again, then.

**Comment by jforissier** [comment]
Updated.

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Pull request merged**